vllm/model_executor/layers/parameters/lazy_compressed.py

```diff
@@ -12,7 +11,7 @@ class LazyCompressedParameter(torch.Tensor):
     def __new__(cls,
                 uncompressed_data: torch.Tensor,
                 storage_format_cls: Type[
-                    CompressedStorageFormat] = SparseBitmaskStorageFormat,
+                    "CompressedStorageFormat"] = "SparseBitmaskStorageFormat",
```

Comment:
Does this work when the model is actually sparse?

I.e. wouldnt we have to import these during the call to `__new__`?

Reply:
yes it is magic, but this isn't land-able since ruff doesn't like it